src/core/components/ErrorPage/index.js

```diff
@@ -2,21 +2,23 @@ import React, { PropTypes } from 'react';
 import { connect } from 'react-redux';
 import { compose } from 'redux';
 
-import { getErrorComponent } from 'amo/utils';
+import { getErrorComponent as getErrorComponentCore } from 'core/utils';
```

Comment:
`getErrorComponentDefault` would be fine (and in line with what we do elsewhere), I don't know why I picked this.